packages/flex-plugin-scripts/src/scripts/pre-script-check.ts

```diff
@@ -195,8 +196,16 @@ const preScriptCheck = async (...args: string[]): Promise<void> => {
   addCWDNodeModule(...args);
 
   _setPluginDir(...args);
+
+  let pkgName = '';
+  try {
+    const packageJson = readPackageJson(join(process.cwd(), 'package.json'));
+    pkgName = packageJson.name;
+  } catch (e) {
+    // no-op
+  }
   const resetPluginDirectory = await checkPluginConfigurationExists(
```

Comment:
What's going to happen if `pkgName` is undefined/empty string?

Reply:
the start script would fail, but it would add another object with an empty string into the plugins.json 🤔  was actually going to ask what is the expected behavior here?

Reply:
hmm should everything from line 207 to line 218 should move into the try block so that it doesn't add to the plugins.json if the name doesn't exist? then i think it would just fail when running the start/build commands